pkg/k8sutil/kotsadm.go

```diff
@@ -203,7 +204,54 @@ func WaitForKotsadm(clientset *kubernetes.Clientset, namespace string, timeoutWa
 		time.Sleep(time.Second)
 
 		if time.Since(start) > timeoutWaitingForWeb {
-			return "", &kotsadmtypes.ErrorTimeout{Message: "timeout waiting for kotsadm pod"}
+			return "", &types.ErrorTimeout{Message: "timeout waiting for kotsadm pod"}
+		}
+	}
+}
+
+func RestartKotsadm(ctx context.Context, clientset *kubernetes.Clientset, namespace string, timeout time.Duration) error {
+	pods, err := clientset.CoreV1().Pods(namespace).List(context.TODO(), metav1.ListOptions{LabelSelector: "app=kotsadm"})
```

Comment:
This use `Delete` here with label selector instead of `List` and then there is `WaitForDeploymentReady` (or `calculateDeploymentState`, not sure which is better).

Reply:
And I just saw this this is existing function that was moved, so this is not new.

Reply:
actually, i'll fix it

Reply:
also, it doesn't look like the `Delete` function accepts label selectors